requirements.txt

```diff
@@ -0,0 +1,5 @@
+awscli
+logbook
+mysql-connector-python
+requests
+sqlalchemy
```

Comment:
Since this is a greenfield project we should probably use pipenv for managing dependencies. It supports generating a requirements.txt file as well, which we will need for using with Salt.

Reply:
I will replace the requirements.txt file with pipenvs Pipfile in the repo